user_docs/en/userGuide.t2t

```diff
@@ -65,46 +72,223 @@ This applies to both original and modified copies of this software, plus any der
 For further details, you can [view the full license. https://www.gnu.org/licenses/old-licenses/gpl-2.0.html]
 For details regarding exceptions, access the license document from the NVDA menu under the "help" section.
 
-+ System Requirements +[SystemRequirements]
-- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2008 R2.
- - For Windows 7, NVDA requires Service Pack 1 or higher.
- - For Windows Server 2008 R2, NVDA requires Service Pack 1 or higher.
-- at least 150 MB of storage space.
--
 
-+ Getting and Setting Up NVDA +[GettingAndSettingUpNVDA]
-If you have not yet got a copy of NVDA, you can download it from the [NV Access website NVDA_URL].
++ NVDA Quick Start Guide +[NVDAQuickStartGuide]
+
+This quick start guide contains three main sections: downloading, initial setup, and running NVDA.
+These are followed by information on adjusting preferences, participating in the community and getting help.
+The information in this guide is condensed from other parts of the [NVDA User Guide ].

```

Comment:
Is this meant to be a link?

```suggestion
The information in this guide is condensed from other parts of the NVDA User Guide.
```

Reply:
I was going to make it a link, especially for when we copy it out of the user guide, but then I wasn't sure how best to do it - perhaps to https://www.nvaccess.org/files/nvda/documentation/userGuide.html and then I was thinking about the other links in this section to other parts of the user guide?

Reply:
I think it's fine to not link it - it is strange for there to be a self-referential link.

This might be better:

```suggestion
The information in this guide is condensed from other parts of this document.
```

Reply:
Should we say "this document", or "this NVDA User Guide" - that way when we do extract it as its own document, someone reading it can at least pick up that we meant the NVDA User Guide not this quick start guide (when we inevitably forget to link it there).